apertium-es-pt.es-pt.crx

```diff
@@ -1,24 +1,35 @@
 <?xml version="1.0"?>
 <capitalization>
   <rules>
-    <rule weight="0.1">
-      <match select="aa"/>
-    </rule>
-    <rule weight="1.0">
+    <rule weight="3.0">
       <or>
-        <match lemma="iPhone" select="dix"/>
-        <match lemma="iPod" select="dix"/>
+        <begin/>
+        <match tags="sent"/>
       </or>
+      <match select="Aa"/>
     </rule>
-    <rule weight="0.9">
+    <rule weight="2.0">
       <match tags="np.*" select="Aa"/>
```

Comment:
the weird words below (iPhone, etc.) would also be proper nouns, I think, so this rule would override that one, giving `Iphone`, `Latex`, etc.